doc/developer-guides/hld/hv-dev-passthrough.rst

```diff
@@ -60,14 +63,39 @@ passthrough, as detailed here:
 
    Passthrough Device Status
 
-DMA Remapping
-*************
+Owner of Passthrough Devices
+****************************
+
+ACRN hypervisor will do PCI enumeration to discover the PCI devices on the platform.
+According to the hypervisor/VM configurations, the owner of these PCI devices can
```

Comment:
I think you're missing part of the sentence here?

Reply:
can "be one of " the ...
will fix the missing word